### PR TITLE
Plots: add /plots/ to gh-pages

### PR DESCRIPTION
**Pull request by wwwillchen**
* Create a static site to view archived runs
* Demo: https://wwwillchen.github.io/lighthouse/plots/

lighthouse-plots-ghpages

### CODE REVIEW

**Comment by paulirish** [comment]
v nice. can you add a readme in here with the basic instructions for how new ones are added?

(pretty sure it's 100% manual, but it'll be nice to confirm this. :)

**Comment by brendankenny** [comment]
> Let people share links to archived runs

yeah, the PR description was a little confusing :) More like it's adding a place for sharing links to archived runs?

**Comment by wwwillchen** [comment]
Added some instructions. Ptal, thanks.

File: plots/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  <meta name="theme-color" content="#304ffe">
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <div class="container">
+    <h1>Lighthouse Plots</h1>
+    <ul>
+      <li>
+        <a href="./archive/040617-70-sites-9-runs-oom/index.html">
```
Comment (paulirish) [inline plots/index.html]: instead of MMDDYY lets use YYYYMMDD

**Comment by wwwillchen** [comment]
Done!

**Pull request merged**